### PR TITLE
[WIP] Progress on review

### PR DESCRIPTION
**Pull request by jpaddison3**

### CODE REVIEW

File: packages/lesswrong/components/review/FrontpageReviewWidget.tsx
Comment (jpaddison3) [inline packages/lesswrong/components/review/FrontpageReviewWidget.tsx:151]: Huh, I'm curious what the issue was?

File: packages/lesswrong/components/review/FrontpageReviewWidget.tsx
Comment (jpaddison3) [inline packages/lesswrong/components/review/FrontpageReviewWidget.tsx:181]:
```suggestion
        <li>Posts with at least one positive vote proceed to the Review Phase.</li>
```

File: packages/lesswrong/components/review/FrontpageReviewWidget.tsx
Comment (jpaddison3) [inline packages/lesswrong/components/review/FrontpageReviewWidget.tsx:183]: This was my bad, why did I think you could put links in a tooltip?

File: packages/lesswrong/components/review/FrontpageReviewWidget.tsx
Comment (jpaddison3) [inline packages/lesswrong/components/review/FrontpageReviewWidget.tsx:223]: I think yearly is actually useful. It shows you the same information, but puts the year at the top, which is helpful orientation for users who aren't used to the review process. But it's your link, your call.

File: packages/lesswrong/components/review/FrontpageReviewWidget.tsx
Comment (jpaddison3) [inline packages/lesswrong/components/review/FrontpageReviewWidget.tsx:244]: Good

File: packages/lesswrong/components/review/ReviewVotingPage.tsx
Comment (jpaddison3) [inline packages/lesswrong/components/review/ReviewVotingPage.tsx:32]: Wait, seriously? You gotta be kidding. Those comparisons have to be the wrong way around. I didn't test it but I was like (internal feeling of) 98% confident. I gotta try. Very apologies if I was wrong.

File: packages/lesswrong/lib/collections/posts/views.ts
Comment (jpaddison3) [inline packages/lesswrong/lib/collections/posts/views.ts:1256]: Why you remove positive? The denormalized field checks for positive-ness.

File: packages/lesswrong/components/review/ReviewVotingPage.tsx
Comment (jpaddison3) [reply packages/lesswrong/components/review/ReviewVotingPage.tsx:32]: Nope, I was right.

Comment (jpaddison3) [reply packages/lesswrong/components/review/ReviewVotingPage.tsx:32]: Asdf looking at the wrong component.

Comment (jpaddison3) [reply packages/lesswrong/components/review/ReviewVotingPage.tsx:32]: I think I had what the function did turned around in my head. :\

File: packages/lesswrong/components/review/FrontpageReviewWidget.tsx
Comment (Raemon) [reply packages/lesswrong/components/review/FrontpageReviewWidget.tsx:183]: I'm not sure we should bother doing this, although I do see the appeal. By the time we get to the review phase we'll already know which posts were overall sorted low, and we'll already need to encourage people to basically ignore low-ranked posts unless they think they're a hidden gem.

File: packages/lesswrong/components/review/FrontpageReviewWidget.tsx
Comment (Raemon) [reply packages/lesswrong/components/review/FrontpageReviewWidget.tsx:239]: The problem is that it also weirdly truncates personal blogposts and emphasizes shortform. I agree with the year being at the top is nice.

**Review by Raemon — APPROVED** [review]

**Review by jpaddison3 — COMMENTED** [review]
I fixed the type errors and lint. ~I'd be interested in discussing the Expanded Post View.~

File: packages/lesswrong/components/review/ReviewVotingExpandedPost.tsx
Comment (jpaddison3) [reply packages/lesswrong/components/review/ReviewVotingExpandedPost.tsx:1]: Maybe you mean to add comments here? Oh there is something that shows unread comments. Nice.

File: packages/lesswrong/components/review/FrontpageReviewWidget.tsx
Comment (jpaddison3) [reply packages/lesswrong/components/review/FrontpageReviewWidget.tsx:239]: What if we set `displayShortform=false`?

I'll add the url query, but I'll let you decide about personal blogposts.

File: packages/lesswrong/components/review/ReviewVotingPage.tsx
Comment (jpaddison3) [reply packages/lesswrong/components/review/ReviewVotingPage.tsx]: Yep, except my pride

File: packages/lesswrong/components/review/FrontpageReviewWidget.tsx
Comment (jpaddison3) [reply packages/lesswrong/components/review/FrontpageReviewWidget.tsx:183]: It already does it tho, check out the denormalized field filter

File: packages/lesswrong/components/review/FrontpageReviewWidget.tsx
Comment (jpaddison3) [inline packages/lesswrong/components/review/FrontpageReviewWidget.tsx:237]: Dumb question: If we're calling this all 2020 posts, why filter for unnominated?

File: packages/lesswrong/lib/collections/posts/views.ts
Comment (jpaddison3) [inline packages/lesswrong/lib/collections/posts/views.ts:98]: In other reason not to filter allposts with this, I'm not confident there's an index supporting this. `TODO`

**Comment by jpaddison3** [comment]
Boy this sure does get annoying:
image
(The hover-over is preventing me from clicking the vote button)

**Comment by jpaddison3** [comment]
I can awkwardly scroll on the reviewVoting page when there aren't that many posts

File: packages/lesswrong/components/review/FrontpageReviewWidget.tsx
Comment (Raemon) [reply packages/lesswrong/components/review/FrontpageReviewWidget.tsx:237]: It should maybe have a different name. Filter for unnominated is so people can focus on adding new posts.

File: packages/lesswrong/components/review/FrontpageReviewWidget.tsx
Comment (Raemon) [reply packages/lesswrong/components/review/FrontpageReviewWidget.tsx]: I think I just twiddled that randomly while trying to fix something, which it didn't help with, but then didn't feel motivated to change it back.

File: packages/lesswrong/components/review/ReviewVotingExpandedPost.tsx
Comment (Raemon) [reply packages/lesswrong/components/review/ReviewVotingExpandedPost.tsx:1]: Part of the point here was just that ReviewVotingPage was huge and unwieldy, and I wanted to break it up into pieces that were easier to think about.

File: packages/lesswrong/lib/collections/posts/views.ts
Comment (Raemon) [reply packages/lesswrong/lib/collections/posts/views.ts:99]: We can just fix that tho

**Review by jpaddison3 — COMMENTED** [review]
We're pulling essentially 24 hour coverage in terms of code pushed to this branch, which is pretty cool 🏎

**Comment by jpaddison3** [comment]
I should say @Raemon, I really like your design sense.

**Comment by jpaddison3** [comment]
I don't love the way the top bar isn't part of the paper?
image

File: packages/lesswrong/server/resolvers/reviewVoteResolvers.ts
Comment (jpaddison3) [inline packages/lesswrong/server/resolvers/reviewVoteResolvers.ts:36]: This is notable, but we have until we turn on quadratic voting to fix it.

File: packages/lesswrong/components/review/FrontpageReviewWidget.tsx
Comment (jpaddison3) [inline packages/lesswrong/components/review/FrontpageReviewWidget.tsx:225]: @Raemon

File: packages/lesswrong/components/review/ReviewPostComments.tsx
Comment (jpaddison3) [inline packages/lesswrong/components/review/ReviewPostComments.tsx:39]: My opinion is that we can fix this later, but we should get to work on it asap.

File: packages/lesswrong/components/review/ReviewVotingExpandedPost.tsx
Comment (jpaddison3) [inline packages/lesswrong/components/review/ReviewVotingExpandedPost.tsx:91]: 👀

Seems fine to push to not-this-PR

File: packages/lesswrong/components/review/ReviewVotingPage.tsx
Comment (jpaddison3) [inline packages/lesswrong/components/review/ReviewVotingPage.tsx:218]: (Later) We can just remove whatever uses this conditional.

File: packages/lesswrong/components/review/ReviewVotingPage2019.tsx
Comment (jpaddison3) [inline packages/lesswrong/components/review/ReviewVotingPage2019.tsx:1]: This page is not working. I think maybe @Raemon you decided that was ok with you? But if not then I think this is the first thing I've seen that would hold up shipping.

File: packages/lesswrong/lib/collections/posts/views.ts
Comment (jpaddison3) [inline packages/lesswrong/lib/collections/posts/views.ts:1278]:
```suggestion
// Nominations for the (≤)2020 review are determined by the number of positive votes
```

File: packages/lesswrong/lib/collections/votes/views.ts
Comment (jpaddison3) [inline packages/lesswrong/lib/collections/votes/views.ts:46]:
```suggestion
```

**Review by jpaddison3 — COMMENTED** [review]
I just reviewed the whole damn thing. There's only one thing that might hold up shipping, on LW's end — that `/reviewVoting/2019` is borked. If @Raemon's fine with that, then :shipit:. Speaking on behalf of the Forum, this is ready to go.

**Pull request merged**
